crates/hyperswitch_domain_models/src/payments/payment_intent.rs

```diff
@@ -133,6 +133,7 @@ pub struct PaymentIntentNew {
     pub session_expiry: Option<PrimitiveDateTime>,
     pub request_external_three_ds_authentication: Option<bool>,
     pub charges: Option<pii::SecretSerdeValue>,
+    pub billing_address_details: Option<Encryptable<Secret<serde_json::Value>>>,
```

Comment:
Can we add these fields to `KafkaPaymentIntent` & `KafkaPaymentIntentEvent` as well ?

Reply:
This is done.